src/main/kotlin/lotto/model/Validator.kt

```diff
@@ -1,48 +1,32 @@
 package lotto.model
 
 class Validator {
+
     fun validateAmount(userAmount: String) {
-        if (userAmount.toIntOrNull() == null) {
-            throw IllegalArgumentException("Input is not a number")
-        }
-        if ((userAmount.toInt() % LOTTO_PRICE) != 0) {
-            throw IllegalArgumentException("Input is not multiple of 1000")
-        }
+        require(userAmount.toIntOrNull() != null) { "Input is not a number" }
+        require((userAmount.toInt() % LOTTO_PRICE) == 0) { "Input is not multiple of 1000" }
+    }
+
+    fun validateNumberOfTickets(input: String, totalNumberOfTickets: Int) {
+        require(input.toIntOrNull() != null) { "Input is not a number" }
+        require(input.toInt() <= totalNumberOfTickets) { "Input is bigger than number of tickets" }
+        require(input.toInt() >= 0) { "Input must be positive number" }
     }
 
-    fun validateWinningNumbers(userInput: List<String>) {
-        userInput.forEach { item ->
-            if (item.toIntOrNull() == null) {
-                throw IllegalArgumentException("Input has invalid numbers")
-            }
-        }
-        if (userInput.count() != 6) {
-            throw IllegalArgumentException("Input should have 6 numbers")
-        }
-        if (userInput.count() != userInput.toSet().count()) {
-            throw IllegalArgumentException("Numbers should be unique")
-        }
-        userInput.forEach { item ->
-            if (item.toInt() !in 1..45) {
-                throw IllegalArgumentException("The winning numbers should be between 1 and 45")
-            }
-        }
+    fun validateTicketNumbers(userInput: List<String>) {
```

Comment:
If it’s only handling input validation, then naming it `InputValidator` would be perfectly appropriate. 😊